docs: update pactflow-ai install instructions / add interactive tutorial (#301)
pactflow/docs.pactflow.io: website/docs/docs/ai/index.md

<AiDownloadTable />

:::note

> Linux gnu users will require glibc version 2.23 or later.

Reviewer: ```suggestion
Linux GNU users will require glibc version 2.23 or later.
```

> For users in distros which a glibc version <2.23 (check by running `ldd --version`), you can download and use the linux musl variant instead.

Reviewer: ```suggestion
Environments which do not use glibc, or use a version of glibc prior to 2.23, should instead use the musl variant.
```
